docs/source/pillars/computing_technology.md

```diff
@@ -264,6 +269,10 @@ _The ability of the TRE organisation to identify, maintain, and verify informati
 * - You must be able to replace a non-compliant TRE with a compliant system.
   - This might involve reconfiguring a running system or by replacing it with a compliant one.
   - Mandatory
+* - You should ensure the TRE automatically applies updates for software used for configuring user or management infrastructure, as well as any anti-virus/malware.
+  - This includes any software used for remote desktop portals, databases, webapps, configuration management or software used for logging within the TRE.
+    Doing so will ensure that the latest software patches are installed and reduce the risk of expoitable vulnerabilities in the TRE infrastructure.
+  - Mandatory
```

Comment:
Also this feels like it's "turtles all the way down" - if you're using a browser on your laptop to configure the infrastructure then this has to be up-to-date, ... but so does the tool that you use to update your browser, ... and so does the OS that you're running on, ... and so does the upstream provider of OS updates etc.

Reply:
Sure, but having a part of the spec that not only says "everything should be updated" for basic IT security reasons but also "here are some of the examples of the TRE components you'd need to be updating" seems like it could be useful? Going by the questions in #60

Reply:
This is one area where we either need the definitions of "User infrastructure" and "Management infrastructure" to make sense, or we don't mention them individually. This also gets ambiguous in the cloud, e.g. if you're using the cloud providers log aggregation service then there's nothing to update.

Reply:
I'm not sure what to do with this one - it seems like we need it to satisfy #60 - would anyone like to suggest or an edit, or should we remove? @manics I've copied the examples of user infrastructure and management infrastructure from that issue, but do we need a clearer definition?